TypeError: Cannot read property 'remoteAddress' of undefined

Hello. Version via `sudo npm install -g thelounge` works fine, but source version cloned in home and installed with `npm install` throws `TypeError`, more below.

This happens as soon as I click connect button.

**System:** Ubuntu server 14.04 LTS
**CLI:** bash
**App/node:** versions below

````
rolle@pulina:~/lounge$ npm start  
  
> thelounge@2.1.0 start /home/rolle/lounge  
> node index  
  
2016-12-12 19:21:24 [INFO] The Lounge source (5213853) is now running using node 7.0.0 on linux (x64)  
2016-12-12 19:21:24 [INFO] Configuration file: /home/rolle/.lounge/config.js  
2016-12-12 19:21:24 [INFO] Available on http://*:9234/ in public mode  
2016-12-12 19:21:24 [INFO] Press Ctrl-C to stop  
  
2016-12-12 19:21:34 [DEBUG] [undefined (#0) on QuakeNet (#0)] Connection.setEncoding() encoding=utf8  
2016-12-12 19:21:34 [DEBUG] [undefined (#0) on QuakeNet (#0)] NetTransport connect()  
2016-12-12 19:21:34 [DEBUG] [undefined (#0) on QuakeNet (#0)] NetTransport disposeSocket() connected=false  
2016-12-12 19:21:34 [DEBUG] [undefined (#0) on QuakeNet (#0)] NetTransport Connection.setEncoding() encoding=utf8  
2016-12-12 19:21:34 [DEBUG] [undefined (#0) on QuakeNet (#0)] NetTransport Connecting socket..  
2016-12-12 19:21:34 [DEBUG] [undefined (#0) on QuakeNet (#0)] NetTransport socketRawConnected()  
2016-12-12 19:21:34 [DEBUG] [undefined (#0) on QuakeNet (#0)] NetTransport socketFullyConnected()  
2016-12-12 19:21:34 [DEBUG] [undefined (#0) on QuakeNet (#0)] Socket fully connected  
/home/rolle/lounge/src/oidentd.js:34  
                        file += "to " + connection.socket.remoteAddress  
                                                         ^  
  
TypeError: Cannot read property 'remoteAddress' of undefined  
    at connections.forEach (/home/rolle/lounge/src/oidentd.js:34:37)  
    at Map.forEach (native)  
    at OidentdFile.refresh (/home/rolle/lounge/src/oidentd.js:33:20)  
    at OidentdFile.addSocket (/home/rolle/lounge/src/oidentd.js:19:8)  
    at IrcClient.<anonymous> (/home/rolle/lounge/src/plugins/irc-events/connection.js:76:33)  
    at IrcClient.emit (/home/rolle/lounge/node_modules/eventemitter3/index.js:149:33)  
    at Connection.<anonymous> (/home/rolle/lounge/node_modules/irc-framework/src/client.js:117:16)  
    at Connection.emit (/home/rolle/lounge/node_modules/eventemitter3/index.js:128:35)  
    at Connection.socketOpen (/home/rolle/lounge/node_modules/irc-framework/src/connection.js:77:14)  
    at emitNone (events.js:86:13)  
  
npm ERR! Linux 3.13.0-57-generic  
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"  
npm ERR! node v7.0.0  
npm ERR! npm  v3.10.8  
npm ERR! code ELIFECYCLE  
npm ERR! thelounge@2.1.0 start: `node index`  
npm ERR! Exit status 1  
npm ERR!  
npm ERR! Failed at the thelounge@2.1.0 start script 'node index'.  
npm ERR! Make sure you have the latest version of node.js and npm installed.  
npm ERR! If you do, this is most likely a problem with the thelounge package,  
npm ERR! not with npm itself.  
npm ERR! Tell the author that this fails on your system:  
npm ERR!     node index  
npm ERR! You can get information on how to open an issue for this project with:  
npm ERR!     npm bugs thelounge  
npm ERR! Or if that isn't available, you can get their info via:  
npm ERR!     npm owner ls thelounge  
npm ERR! There is likely additional logging output above.  
  
npm ERR! Please include the following file with any support request:  
npm ERR!     /home/rolle/lounge/npm-debug.log  
````

````
rolle@pulina:~/lounge$ cat npm-debug.log 
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'start' ]
2 info using npm@3.10.8
3 info using node@v7.0.0
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle thelounge@2.1.0~prestart: thelounge@2.1.0
6 silly lifecycle thelounge@2.1.0~prestart: no script for prestart, continuing
7 info lifecycle thelounge@2.1.0~start: thelounge@2.1.0
8 verbose lifecycle thelounge@2.1.0~start: unsafe-perm in lifecycle true
9 verbose lifecycle thelounge@2.1.0~start: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:/home/rolle/lounge/node_modules/.bin:/home/rolle/bin:/opt/sp/bin:/opt/sp/php5.6/bin:/opt/sp/apache/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
10 verbose lifecycle thelounge@2.1.0~start: CWD: /home/rolle/lounge
11 silly lifecycle thelounge@2.1.0~start: Args: [ '-c', 'node index' ]
12 silly lifecycle thelounge@2.1.0~start: Returned: code: 1  signal: null
13 info lifecycle thelounge@2.1.0~start: Failed to exec start script
14 verbose stack Error: thelounge@2.1.0 start: `node index`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:877:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
15 verbose pkgid thelounge@2.1.0
16 verbose cwd /home/rolle/lounge
17 error Linux 3.13.0-57-generic
18 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
19 error node v7.0.0
20 error npm  v3.10.8
21 error code ELIFECYCLE
22 error thelounge@2.1.0 start: `node index`
22 error Exit status 1
23 error Failed at the thelounge@2.1.0 start script 'node index'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the thelounge package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     node index
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs thelounge
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls thelounge
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
````

--- a/src/plugins/irc-events/connection.js
+++ b/src/plugins/irc-events/connection.js
@@ -64,16 +64,16 @@ module.exports = function(irc, network) {
 	});
 
 	if (identd.isEnabled()) {
-		irc.on("socket connected", function() {
-			identd.hook(irc.connection.socket, client.name || network.username);
+		irc.on("raw socket connected", function(socket) {
+			identd.hook(socket, client.name || network.username);
 		});
 	}
 
 	if (identHandler) {
 		let identSocketId;
 
-		irc.on("socket connected", function() {
-			identSocketId = identHandler.addSocket(irc.connection.socket, client.name || network.username);
+		irc.on("raw socket connected", function(socket) {
+			identSocketId = identHandler.addSocket(socket, client.name || network.username);
 		});
 
 		irc.on("socket close", function() {

